### PR TITLE
Fix Move Package

### PR DESCRIPTION
**Pull request by as3boyan**
Fix Move Package

### CODE REVIEW

**Comment by sganapavarapu1** [comment]
Code changes make sense.
Apart from checking `testMovePackage` result, I'm assuming you also did functional testing of MovePackage feature in a sample test project (from `testData` folder) and validated.
Approved for merge.

File: src/com/intellij/plugins/haxe/util/HaxeResolveUtil.java [hunk 1]
```diff
@@ -578,21 +578,26 @@ public static String getQName(@NotNull PsiElement type, boolean searchInSamePack
       return usingStatement.getReferenceExpression().getText();
     }
 
-    String text = type.getText();
-    if (type instanceof HaxeReferenceExpression) {
```
Comment (as3boyan) [inline src/com/intellij/plugins/haxe/util/HaxeResolveUtil.java]: Seems like it was a bad idea to add this before next part. (https://github.com/TiVo/intellij-haxe/commit/4b822108dc322edc122dfe06aa34e37b5b081253)
https://github.com/TiVo/intellij-haxe/issues/39

**Comment by as3boyan** [comment]
I think we need test case for https://github.com/TiVo/intellij-haxe/issues/39, then I can tell is this commit good enough, because breaking change was caused by my implementation for #39

**Comment by as3boyan** [comment]
Basically here I just moved down implementation for #39( now has lower priority)

**Comment by sganapavarapu1** [comment]
yup, do you want to add `testMovePackage2` (or something) that tests #39

**Comment by as3boyan** [comment]
I would prefer more isolated test case, which just has two files (different packages) and attempts to resolve some public static member from class in another file

**Comment by as3boyan** [comment]
Move package, seem to work fine, I think, when I tried to make same changes to latest, I wasn't able to get it to work.

**Comment by EBatTiVo** [comment]
What's the status on this pull request, guys?

**Comment by as3boyan** [comment]
I was unable to merge this yet.

**Comment by as3boyan** [comment]
Passes test for #39
https://github.com/TiVo/intellij-haxe/pull/256

**Comment by as3boyan** [comment]
Merged in #256

**Pull request closed without merging**